src/main/groovy/com/fidesmo/gradle/plugin/FidesmoPlugin.groovy

```diff
@@ -82,29 +82,35 @@ class FidesmoPlugin implements Plugin<Project> {
             }
         }
 
-        project.tasks.create('installToLocalCard', OperationTask) {
-            group = 'publish'
-            description = 'Installs the executable load file to fidesmo card via a locally attached card reader'
-            dependsOn(project.uploadExecutableLoadFile)
-            dependsOn(project.deleteFromLocalCard)
-
-            doLast {
-                def applicationAid
-                if(fidesmoExtension?.instanceAid) {
-                    applicationAid = new Aid(fidesmoExtension.instanceAid).hexString
-                } else {
-                    applicationAid = jcExtension.cap.applets.first().aid.hexString
+        def installToLocalCard = { encryptLoad ->
+            { it ->
```

Comment:
The `it` parameter is always implicit unless you write `{ -> ... }`. It exists for iterator cases. Maybe we should name this parameter more descriptive (like `task`).